modules/exploits/windows/misc/crosschex_device_bof.rb

```diff
@@ -0,0 +1,81 @@
+
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = NormalRanking
+  PAYLOAD_MAX_LENGTH = 8947
+  PACKET_LEN = 10
+
+  include Msf::Exploit::Remote::Udp
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name'        => 'Anviz CrossChex Local Buffer Overflow',
+      'Description'	=> %q{
+        Waits for broadcasts from Ainz CrossChex looking for new devices, and returns a custom broadcast,
+        triggering buffer overflow.
+      },
+      'Author'	  	=>
+        [
+            'Luis Catarino <lcatarino@protonmail.com>',  # original discovery/exploit
+            'Pedro Rodrigues <pedrosousarodrigues@protonmail.com>',   # original discovery/exploit
+            'agalway-r7',  # Module creation
+            'adfoster-r7' # Module creation
+        ],
+      'License'		  => MSF_LICENSE,
+      'References'	=>
+        [
+            ['CVE', "2019-12518"],
+            ['URL', "https://www.0x90.zone/multiple/reverse/2019/11/28/Anviz-pwn.html"],
+            ['EDB', "47734"]
+        ],
+      'DefaultOptions' =>
+        {
+            'TIMEOUT' => 100,
+            'EncoderType' => Msf::Encoder::Type::Raw,
+            'CHOST' => "0.0.0.0",
+            'CPORT' => 5050
+        },
+      'Privileged'	=> true,
+      'Platform' => 'win',
```

Comment:
It would be helpful to specify `Arch`, too, even though there's a default.

Reply:
Added